cmd/uptest/main.go

```diff
@@ -54,8 +54,9 @@ var (
 	defaultConditions = e2e.Flag("default-conditions", "Comma separated list of default conditions to wait for a successful test.\n"+
 		"Conditions could be overridden per resource using \"uptest.upbound.io/conditions\" annotation.").Default("Ready").String()
 
-	skipDelete = e2e.Flag("skip-delete", "Skip the delete step of the test.").Default("false").Bool()
-	testDir    = e2e.Flag("test-directory", "Directory where kuttl test case will be generated and executed.").Envar("UPTEST_TEST_DIR").Default(filepath.Join(os.TempDir(), "uptest-e2e")).String()
+	skipDelete               = e2e.Flag("skip-delete", "Skip the delete step of the test.").Default("false").Bool()
+	testDir                  = e2e.Flag("test-directory", "Directory where kuttl test case will be generated and executed.").Envar("UPTEST_TEST_DIR").Default(filepath.Join(os.TempDir(), "uptest-e2e")).String()
+	onlyCleanUptestResources = e2e.Flag("only-clean-uptest-resources", "While deletion step, only clean resources that were created by uptest").Default("false").Bool()
```

Comment:
Should it be `Default("false")` ? It is more obvious behavior, especially for the newcomer to uptest framework. Can help people to avoid destroying resources accidentally with uptest :)

Reply:
I think it makes sense and it seems more reliable, too. However, this will be not a backward compatible change and effects all Uptest used repos. Because of this I wanted to preserve this behavior.

Reply:
got it, thank you